packages/a2aprotocol/README.md

```diff
@@ -8,6 +8,35 @@
         <img src="https://img.shields.io/pypi/dw/microsoft-teams-a2a" />
     </a>
 </p>
+
+Agent-to-Agent (A2A) protocol support for Microsoft Teams AI applications.
+Enables Teams agents to communicate and collaborate with other AI agents using standardized protocols.
+
 <a href="https://microsoft.github.io/teams-ai" target="_blank">
     <img src="https://img.shields.io/badge/📖 Getting Started-blue?style=for-the-badge" />
 </a>
+
+## Installation
+
+```bash
+uv add microsoft-teams-a2a
+```
+
+## Usage
```

Comment:
Show a simple use case of a2a server (like you already are). And also using a2a clients